lib/v2/audiobar/latest.js

```diff
@@ -0,0 +1,52 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const iconv = require('iconv-lite');
+const { parseDate } = require('@/utils/parse-date');
+
+const rootUrl = 'http://www.audiobar.cn/';
+
+module.exports = async (ctx) => {
+    const currentUrl = `${rootUrl}/all.php`;
+
+    const response = await got({
+        method: 'get',
+        url: currentUrl,
+        responseType: 'buffer',
+    });
+    const data = iconv.decode(response.data, 'gb2312');
+    const $ = cheerio.load(data);
+
+    const list = $('.thread_every_container_div')
+        .map((_, item) => {
+            item = $(item);
+
+            const link = item.find('.thread_subject').attr('href');
+            const dateStr = item.find('.atrd_span').text().slice(-26).replace(' ', '');
+            return {
+                title: item.find('.thread_subject').text().replace('\n', '').replace(' ', ''),
+                link: link.indexOf('http') > -1 ? link : `${rootUrl}${link}`,
+                pubDate: parseDate(dateStr, 'YY-M-D H:mm'),
```

Comment:
```suggestion
                pubDate: timezone(parseDate(dateStr, 'YY-M-D HH:mm'), +8),
```
Missing timezone adjustment